packages/grafana-ui/src/components/Tags/Tag.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { cx, css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
+import { getTagColorsFromName } from '../../utils';
+
+interface TagProps {
+  /** Name of the tag to display */
+  name: string;
+  onClick?: (name: string) => any;
+  /** Custom styles for the wrapper component */
+  className?: string;
+  [key: string]: any;
+}
+
+export const Tag: FC<TagProps> = ({ name, onClick, className, ...rest }) => {
```

Comment:
Don't know if we need to pass some other props to this component, but I was thinking this could be used for smth like `area-label`, etc.

Reply:
It's a valid point, I think it doesn't do any harm in this component since it's a `span`

Reply:
See @torkelo's comment, it's probably a better solution